clash-ghc/src-ghc/Clash/GHC/Evaluator/Primitive.hs

```diff
@@ -2235,6 +2398,12 @@ ghcPrimStep tcm isSubj pInfo tys args mach = case primName pInfo of
            val = unpack (toBV i :: BitVector 64)
         in reduce (mkDoubleCLit tcm val resTy)
 
+  "Clash.Class.BitPack.Internal.unpackCUShort#"
+    | [(_msk, i)] <- bitVectorLiterals' args
```

Comment:
@reviewer What kind of an error to generate of bits are undefined? Something like

```
Cannot unpack bitvector <lit> because of undefined bits?
```